src/domains/my/detail/components/MembershipPwCheck.jsx

```
@@ -0,0 +1,115 @@
+import { useNavigate } from 'react-router-dom'
+import { useForm } from 'react-hook-form'
+import Button from '../../../../components/web/Button'
+import { postPWCheck } from '../../services/userMyDataServices'
+import { useState } from 'react'
+import useAuthStore from '../../../../store/login/useAuthStore'
+import { toast } from 'react-toastify'
+import PwInputBox from '../../../../components/web/PwInputBox'
+
+const MembershipPwCheck = ({ isOpen, onClose, onButtonClick, onSuccessAction }) => {
```

ESLint

[error] 'isOpen' is missing in props validation

(react/prop-types)